src/CalcManager/CalcManager.vcxproj

```diff
@@ -346,4 +348,4 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets" />
-</Project>
+</Project>
```

Comment:
Please add an empty line following the last non-empty line.
Same for the other files.